Healex.HL7v2Anonymizer.Tests/AnonymizerTests.cs

```diff
@@ -1,108 +1,337 @@
 using Healex.HL7v2Anonymizer.Services;
 using HL7.Dotnetcore;
-using Microsoft.Extensions.Configuration;
 using Microsoft.VisualStudio.TestTools.UnitTesting;
-using System;
-using System.IO;
 using System.Linq;
+using FluentAssertions;
 using static Healex.HL7v2Anonymizer.ReplacementOptions;
 
-namespace Healex.HL7v2Anonymizer.Tests {
-
+namespace Healex.HL7v2Anonymizer.Tests
+{
     [TestClass]
-    public class AnonymizerTests {
-
+    public class AnonymizerTests
+    {
+        #region helperObjects
+        private Anonymizer ruleExpander;
+        #endregion
+        
         #region Test Methods
 
         [TestMethod]
-        [DeploymentItem("Healex.HL7v2Anonymizer.Tests/TestData", "TestData")]
-        public void AnonymizerTestAdt() {
-            TestAnonymization(File.ReadAllText(@"./TestData/TestAdt1.hl7"));
+        public void Anonymize_NoRepetitionPid5_PatientNameNotInMessage()
+        {
+            var testMessage = CreateDefaultTestMessage(0);
+            var testReplacements = CreateDefaultReplacementOptions();
+            
+            var anonymizer = new Anonymizer(testReplacements);
+            anonymizer.Anonymize(testMessage);
+            var x = testMessage.SerializeMessage(false);
+            x.Should().NotContain("Mustermann").And.NotContain("Maximilian");
         }
+        
+        [TestMethod]
+        public void Anonymize_RepetitionPid5_PatientNameNotInMessage()
+        {
+            var testMessage = CreateDefaultTestMessage(4);
+            var testReplacements = CreateDefaultReplacementOptions();
+            
+            var anonymizer = new Anonymizer(testReplacements);
+            anonymizer.Anonymize(testMessage);
 
+            var x = testMessage.SerializeMessage(false);
+            x.Should().NotContain("Mustermann").And.NotContain("Maximilian");
+        }
+        
         [TestMethod]
-        [DeploymentItem("Healex.HL7v2Anonymizer.Tests/TestData", "TestData")]
-        public void AnonymizerTestOru() {
-            TestAnonymization(File.ReadAllText(@"./TestData/TestOru1.hl7"));
+        public void AnonymizerTestAdt()
+        {
+            TestAnonymization(CreateDefaultTestMessage(0));
         }
 
         [TestMethod]
-        [DeploymentItem("Healex.HL7v2Anonymizer.Tests/TestData", "TestData")]
-        public void AnonymizerTestAdtSegmentOrderWithAnonymization() {
+        public void AnonymizerTestOru()
+        {
+            TestAnonymization(CreateDefaultTestMessage(0));
+        }
```

Comment:
After removing the ORU test message, this is test is redundant as it is exactly the same as `AnonymizerTestAdt`.